packages/composite-checkout/src/components/checkout-step.js

```diff
@@ -23,6 +23,7 @@ export default function CheckoutStep( {
 	finalStep,
 	stepSummary,
 	stepContent,
+	editButtonAriaLabel,
```

Comment:
IIRC the `*AriaLabel` props came in originally to help with testing. It's not crucial to do anything about this, but is this name for that prop too ambiguous? Reading it now I'm not sure if this is the _text_ or the _id_ of the aria label.

Reply:
🤔 Thanks for the info. Hm... well, if it's not important we could change it so that the edit buttons all share a generic label, which would make it so we don't need this prop at all. Something like "Edit this step". What do you think? Would that break tests? Does it defeat the purpose of an aria-label?

Reply:
This could also be changed in a later PR if we are not sure.